integration-tests/interactions/bot/daily_ci_enroll/thank_you.jinja

```diff
@@ -0,0 +1,7 @@
+{% import 'bot/macros.jinja' as bot %}
+
+[
+    {
+        "text": "Thanks and take care of yourself!"
+    }
+]
```

Comment:
This thank you message is generic, it should probably be at the root of the bot folder.

Reply:
Got it!